rockcraft-pack/action.yml

```diff
@@ -14,6 +14,12 @@ inputs:
 
       The default is 'trace'.
     default: 'trace'
+  revision:
+    description: >
+      Pin the snap revision to install.
+
+      If not provided, it defaults to whatever revision is in latest/stable.
```

Comment:
I would say this use is potentially not recommended as there is no easy update strategy.

Reply:
Can you elaborate? What's not recommended? Pinning or using the `latest/stable`?

Reply:
I do not recall :shrug: